roles/neutron-data-network/tasks/main.yml

```diff
@@ -110,6 +117,7 @@
   when: neutron.lbaas.enable
 
 - include: ipchanged.yml
+  when: not neutron.l3ha.enabled|bool
```

Comment:
Using neutronclient python library to check if l3 ha is enabled. If enabled, do not run migrate  neutron services? So far I did not find any function to check if l3 ha is enabled. I will continue to check the neutron client python library.